test/Colors.test.js

```diff
@@ -0,0 +1,89 @@
+const COLORS = require("../src/Colors");
+const assert = require("assert");
+
+test("Colors Test", () => {
+	const color = COLORS.randomColor();
+	color.forEach(elem => {
+		assert.equal(typeof elem, "number");
+	});
+});
+
+test("should return the same color three", () => {
```

Comment:
These tests will fail once we start adding new colors, or we'd have to update these to basically hard code all but one color in each test. 

Maybe these tests can confirm that the result is _not_ in the exclusion list instead of one specific color?

Reply:
You could also do something like set exclusion list to be equal to the color list and then prune what you want, then confirm that the result of the function call is in the stuff you pruned